scipy-1.0/covlett.tex

```diff
@@ -5,70 +5,89 @@
 \topmarginskip{0in}
 \headermarginsize{0in}
 \headermarginskip{0in}
+\MinHead{0in}
+%\dateskipafter{0pt}
+%\dateskipbefore{0pt}
+%\sigskipbefore{0pt}
+%\sigskipafter{0pt}
+\closeskipbefore{1pt}
+%\closeskipafter{0pt}
+%\addrfromskipbefore{0pt}
+\addrfromskipafter{0in}
+\addrtoskipbefore{0in}
+%\addrtoskipafter{0pt}
+\newlfmP{Headlinewd=0pt,Footlinewd=0pt}
+\newlfmP{sigsize=10pt}
+\bottommarginskip{0pt}
+\MinFoot{0pt}
 
 \usepackage{kpfonts}
 \usepackage{url}
 
 \widowpenalty=1000
 \clubpenalty=1000
 
-\newlfmP{headermarginskip=10pt}
-\newlfmP{sigsize=30pt}
-\newlfmP{dateskipafter=20pt}
-%\newlfmP{addrfromphone}
-%\newlfmP{addrfromemail}
-%\PhrPhone{Phone}
-%\PhrEmail{Email}
-
 \namefrom{Ralf Gommers,
           Matt Haberland,
           and Tyler Reddy}
 \addrfrom{%
     \today\\[10pt]
+    Ralf Gommers,\\
     Quansight Labs,\\
-    The Netherlands\\
+    The Netherlands,\\
+    \texttt{ralf.gommers@gmail.com},\\
+    %phone number\\
     \\
+    Matt Haberland,\\
     BioResource and Agricultural Engineering,\\
     California Polytechnic State University,\\
-    San Luis Obispo, CA, 93407, USA\\
+    San Luis Obispo, CA, 93407, USA,\\
+    \texttt{mhaberla@calpoly.edu},\\
+    %phone number\\
     \\
+    Tyler Reddy,\\
     CCS-7 Applied Computer Science Group,\\
     Los Alamos National Laboratory,\\
-    Los Alamos, NM, 87545, USA
+    Los Alamos, NM, 87545, USA,\\
+    \texttt{treddy@lanl.gov},\\
+    %phone number\\
 }
 
 \addrto{%
-Prof. Syma Khalid,\\
-School of Chemistry,\\
-University of Southampton,\\
-Highfield Campus,\\
-Southampton,\\
-SO17 1BJ, U.K.}
+Dr. Rita Strack,\\
+Nature Methods Editorial Office,\\
+One New York Plaza Suite 4500,\\
+New York, NY 10004,\\
+USA}
 
-\greetto{Dear Prof. Khalid,}
+\greetto{Dear Dr. Strack,}
 \closeline{Sincerely,}
 \begin{document}
 \begin{newlfm}
 
 We are submitting the enclosed manuscript describing a milestone
 1.0 release of SciPy for consideration to be published in
-\emph{Scientific Reports}. SciPy is the fundamental scientific 
+\emph{Nature Methods}, as informally discussed.
+SciPy is the fundamental scientific 
 computing library in the Python programming language. 
 Hundreds of experts have contributed to our library, which
 has been downloaded millions of times and has over 100,000 dependent
-code repositories.
+code repositories, including many used in the life sciences.
 
-We have specifically chosen \emph{Scientific Reports} because
-the breadth of acceptable topics matches the variety of algorithms
-our library provides. High-profile manuscripts citing SciPy range
-from the biosciences to recent reports on gravitational waves and
+We have specifically chosen \emph{Nature Methods} because
+SciPy lies at the base of the entire ecosystem of scientific methods
+described in the journal scope, and because SciPy is effectively an
+ever-improving methodological library. High-profile manuscripts citing SciPy 
+extend beyond the biosciences to recent reports on gravitational waves and
```

Comment:
I was going to suggest adding support that SciPy is used in the biosciences. 
"Among the 3000+ articles that cite the SciPy library are hundreds in the field of biology, including several in Nature Methods and other publications by Nature."

Two in Nature Methods:
https://www.nature.com/articles/s41592-018-0114-z
https://www.nature.com/articles/s41592-019-0403-1

Even though not all of the papers in this [quick search](https://scholar.google.com/scholar?hl=en&as_sdt=2005&sciodt=0%2C5&cites=2086009121748039507&scipsc=1&q=biology&btnG=) are actually bioscience-focused, many of them certainly are.

But still, I'm not sure we can hit this criterion:
"To enhance the practical relevance of each paper, description of the method must be accompanied by its validation, its application to an important biological question and results illustrating its performance in comparison to available approaches."

Reply:
thanks @mdhaber, those are useful references. from the [NSF proposal](https://figshare.com/articles/Mid-Scale_Research_Infrastructure_-_The_Scientific_Python_Ecosystem/8009441): _Qiime [6] for gene sequencing (13,500 citations)_ is the most high-profile bio tool we could find

> But still, I'm not sure we can hit this criterion:

agreed, but not much we can do about it i'm afraid ....

Reply:
Yeah I figure lets just send it, if we don't have to reformat, and see how it goes.

Reply:
I've revised the cover letter based on the feedback above & to reflect the availability of the preprint.

Let me know what you think. Sadly, we run over to a second page now, but maybe that's ok for the extra clarification of relevance. I've somewhat-awkwardly used footnotes for a few citations, but maybe good enough.

Reply:
Do you think it could help adding those reference and that sentence in the introduction along with the black hole one?

Reply:
@apbard Maybe, though I'm not sure we want to drag on paper adjustments too much more at this stage. I'll mark this discussion as resolved but feel free to open an issue